### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by kuzivka**

### CODE REVIEW

File: src/components/modal/TaskViewModal.tsx [hunk 1]
```diff
@@ -12,6 +12,22 @@ interface ITaskViewModalProps {
   initialTask?: ToDo;
 }
 
+const stylesForModal = {
```
Comment (Yuliia143) [inline src/components/modal/TaskViewModal.tsx]: Can't you create separate file for styles and import this object here?

Comment (kuzivka) [reply src/components/modal/TaskViewModal.tsx]: okay

File: src/components/cards/TaskCards.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ export default function TaskCards() {
 
   return (
     <>
-      <List className="tasks-list">
+      <List sx={{ width: 600, maxWidth: '80vw', m: 'auto' }}>
```
Comment (Kost927) [inline src/components/cards/TaskCards.tsx:18]: you can make a separate file for styles, where you can keep your objects with styles and export them here.

File: src/components/cards/TaskInput.tsx [hunk 3]
```diff
@@ -58,25 +61,32 @@ export default function TaskInput() {
   };
 
   return (
-    <div className="task-input-container">
-      <Sort className="sort-icon" onClick={togleOpenSortingOptins} />
+    <Box
+      sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}
```
Comment (Kost927) [inline src/components/cards/TaskInput.tsx:65]: same

File: src/components/cards/TaskInput.tsx [hunk 4]
```diff
@@ -58,25 +61,32 @@ export default function TaskInput() {
...
+          borderRadius: '3px',
+        }}
+        onClick={togleOpenSortingOptins}
+      />
       <Select
         open={isOptionsOpen}
         onClose={togleOpenSortingOptins}
         onOpen={togleOpenSortingOptins}
-        className="select-input"
+        sx={{ visibility: 'hidden', width: 0, mr: 1 }}
```
Comment (Kost927) [inline src/components/cards/TaskInput.tsx:79]: same, and below

**Pull request merged**